[CI] RcsCcsCommonYamlTestSuiteIT class failing

**Build Scans:**
- [elasticsearch-periodic #8811 / release-tests](https://gradle-enterprise.elastic.co/s/qdya2rqhbmgtu)
- [elasticsearch-pull-request #75784 / part-6](https://gradle-enterprise.elastic.co/s/nexn2ce5adapg)

**Reproduction Line:**
```
./gradlew ":qa:ccs-common-rest:yamlRestTest" --tests "org.elasticsearch.test.rest.yaml.RcsCcsCommonYamlTestSuiteIT" -Dtests.method="test {p0=search.vectors/46_knn_search_bbq_ivf/default oversample value}" -Dtests.seed=7F5CB27BF81D5110 -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dlicense.key=x-pack/license-tools/src/test/resources/public.key -Dtests.locale=pt-MZ -Dtests.timezone=Egypt -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.test.rest.yaml.RcsCcsCommonYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(org.elasticsearch.test.rest.yaml.RcsCcsCommonYamlTestSuiteIT),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [search.vectors/46_knn_search_bbq_ivf:7]: expected [2xx] status code but api [indices.create] returned [400 Bad Request] [---
error:
  root_cause:
  - type: "mapper_parsing_exception"
    reason: "Unknown vector index options type [bbq_ivf] for field [vector]"
    stack_trace: "org.elasticsearch.index.mapper.MapperParsingException: Unknown vector\
      \ index options type [bbq_ivf] for field [vector]\n\tat org.elasticsearch.server@9.1.0/org.elasticsearch.index.mapper.vectors.DenseVectorFieldMapper.parseIndexOptions(DenseVectorFieldMapper.java:2802)\n\
      \tat org.elasticsearch.server@9.1.0/org.elasticsearch.index.mapper.vectors.DenseVectorFieldMapper$Builder.lambda$new$15(DenseVectorFieldMapper.java:309)\n\
      \tat org.elasticsearch.server@9.1.0/org.elasticsearch.index.mapper.FieldMapper$Parameter.parse(FieldMapper.java:973)\n\
      \tat org.elasticsearch.server@9.1.0/org.elasticsearch.index.mapper.FieldMapper$Builder.parse(FieldMapper.java
[truncated]
```

**Issue Reasons:**
- [main] 2 failures in class org.elasticsearch.test.rest.yaml.RcsCcsCommonYamlTestSuiteIT (0.7% fail rate in 306 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/qa/ccs-common-rest/src/yamlRestTest/java/org/elasticsearch/test/rest/yaml/CcsCommonYamlTestSuiteIT.java
+++ b/qa/ccs-common-rest/src/yamlRestTest/java/org/elasticsearch/test/rest/yaml/CcsCommonYamlTestSuiteIT.java
@@ -90,7 +90,8 @@ public class CcsCommonYamlTestSuiteIT extends ESClientYamlSuiteTestCase {
         // geohex_grid requires gold license
         .setting("xpack.license.self_generated.type", "trial")
         .feature(FeatureFlag.TIME_SERIES_MODE)
-        .feature(FeatureFlag.SUB_OBJECTS_AUTO_ENABLED);
+        .feature(FeatureFlag.SUB_OBJECTS_AUTO_ENABLED)
+        .feature(FeatureFlag.IVF_FORMAT);
 
     private static ElasticsearchCluster remoteCluster = ElasticsearchCluster.local()
         .name(REMOTE_CLUSTER_NAME)

--- a/qa/ccs-common-rest/src/yamlRestTest/java/org/elasticsearch/test/rest/yaml/RcsCcsCommonYamlTestSuiteIT.java
+++ b/qa/ccs-common-rest/src/yamlRestTest/java/org/elasticsearch/test/rest/yaml/RcsCcsCommonYamlTestSuiteIT.java
@@ -92,6 +92,7 @@ public class RcsCcsCommonYamlTestSuiteIT extends ESClientYamlSuiteTestCase {
         .setting("xpack.security.remote_cluster_client.ssl.enabled", "false")
         .feature(FeatureFlag.TIME_SERIES_MODE)
         .feature(FeatureFlag.SUB_OBJECTS_AUTO_ENABLED)
+        .feature(FeatureFlag.IVF_FORMAT)
         .user("test_admin", "x-pack-test-password");
 
     private static ElasticsearchCluster fulfillingCluster = ElasticsearchCluster.local()

